attestation/BUILD

```diff
@@ -27,6 +27,9 @@ rust_library(
         "@salus-index//:hex",
         "@salus-index//:typenum",
     ],
+    rustc_flags = [
+        "-Cforce-frame-pointers",
+    ],
```

Comment:
instead of putting this in every single `BUILD` file, could you put it in the `.bazelrc` or similar?

Reply:
Well examine

Reply:
Fixed. added to .bazelrc